### PR TITLE
User list test

### PR DESCRIPTION
**Pull request by MilanPospisil**
Test sorting, paging, filtering for user list.

### CODE REVIEW

File: test/cypress/integration/user_list.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    items = sortBy(items, 'name');
+  });
+
+  beforeEach(() => {
+    cy.login(adminUsername, adminPassword);
+    cy.visit('/ui/users');
+  });
+
+  it('table contains all columns', () => {
+    cy.contains('Username');
```
Comment (himdel) [inline test/cypress/integration/user_list.js:31]: same issues as in https://github.com/ansible/ansible-hub-ui/pull/815/files#r698711313, this needs to look in the headers, not the whole page

Comment (himdel) [reply test/cypress/integration/user_list.js:31]: What about something like...

```js
['Username', 'First name', 'Last name', 'Email', 'Groups', 'Created'].forEach((column) => 
  cy.contains('tr[aria-labelledby="headers"] th', column)
);
```

Comment (MilanPospisil) [reply test/cypress/integration/user_list.js:31]: Good idea.

Comment (himdel) [reply test/cypress/integration/user_list.js:31]: :+1: you may still want to remove the first version ;)

(and maybe rebase, now that #815 is in, for the sort-table change)

Comment (MilanPospisil) [reply test/cypress/integration/user_list.js:31]: Ok.

File: test/cypress/integration/user_list.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    cy.contains('First name');
+    cy.contains('Last name');
+    cy.contains('Email');
+    cy.contains('Groups');
+    cy.contains('Created');
+
```
Comment (himdel) [inline test/cypress/integration/user_list.js:37]:
```suggestion
```

**Pull request merged**